helix-term/src/commands/engine/steel.rs

```diff
@@ -871,7 +871,10 @@ fn load_editor_api(engine: &mut Engine, generate_sources: bool) {
         let mut target_directory = helix_runtime_search_path();
 
         if !target_directory.exists() {
-            std::fs::create_dir(&target_directory).unwrap();
+            std::fs::create_dir_all(&target_directory).unwrap_or_else(|err| {
+                panic!("Failed to create directory {:?}: {}", target_directory, err)
+            });
+            eprintln!("Created directory: {:?}", target_directory);
```

Comment:
I think that both if these are unlikely cases.  Most users have a dir `~/.config`, so they'll never see these.  I was in a build sandbox where those files didn't exist and it wasn't obvious why the file generation was failing or where the generated files were ending up. These messages helped me, I figured I'd leave them here just in case others end up with the same confusion.